src/main/java/flab/just10minutes/member/domain/Member.java

```diff
@@ -0,0 +1,21 @@
+package flab.just10minutes.member.domain;
+
+import lombok.*;
+
+import java.util.Optional;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor
+public class Member {
+
+    @Getter private String id;
+    private String password;
+    @Getter private String name;
+    @Getter private String address;
+    @Getter private Long balance;
```

Comment:
https://www.baeldung.com/lombok-omit-getter-setter#1-overriding-getter-and-setter-annotations